readme.txt

```diff
@@ -3,7 +3,7 @@ Plugin URI: https://github.com/PhazeRoOman/thawani-for-woocommerce
 Author: PhazeRo
 Author URI: https://phaze.ro/
 Description: Thawani Payments Gateway for Woocommerce.
-Version: 1.3.0
+Version: 1.4.0
```

Comment:
@DevAseel  I'd make it 1.4.1 since _tag_ `1.4.0` is released. So we can later create a new tag and dump the update! thanks for taking the initiative

Reply:
@w7shdev Sweet! Made the changes and created a [new tag](https://github.com/PhazeRoOman/thawani-for-woocommerce/tags).